bumble/controller.py

```diff
@@ -895,6 +906,35 @@ def on_classic_sco_connection_complete(
             )
         )
 
+    def on_remote_name_request(self, peer_address: hci.Address, name_offset: int):
+        self.send_lmp_packet(
+            peer_address,
+            lmp.LmpNameRes(
+                name_offset=name_offset,
+                name_length=len(self.local_name.encode('utf-8')),
+                name_fregment=self.local_name.encode('utf-8')[name_offset],
+            ),
+        )
+
+    def on_remote_name_respond(
```

Comment:
Handlers should be called `on_[noun]`
```suggestion
    def on_remote_name_response(
```